src/Symfony/Bundle/SecurityBundle/Tests/Functional/UserPasswordEncoderCommandTest.php

```diff
@@ -71,11 +71,25 @@ public function testEncodePasswordNoConfigForGivenUserClass()
         $this->passwordEncoderCommandTester->execute(array(
             'command' => 'security:encode-password',
             'password' => 'password',
-            'user-class' => 'Wrong/User/Class',
-            'salt' => 'AZERTYUIOPOfghjklytrertyuiolnbcxdfghjkytrfghjk',
+            '--user-class' => 'Wrong/User/Class',
+            '--salt' => 'AZERTYUIOPOfghjklytrertyuiolnbcxdfghjkytrfghjk',
         ));
     }
 
+    public function testEncodePasswordWithNoSaltNoInteraction()
+    {
+        $this->passwordEncoderCommandTester->execute(
+            array(
+                'command' => 'security:encode-password',
+                'password' => 'password',
+                '--user-class' => 'Symfony\Component\Security\Core\User\User',
+            ),
+            array('interactive' => false)
+        );
+
+        $this->assertRegExp('/Password encoding succeeded/', $this->passwordEncoderCommandTester->getDisplay());
```

Comment:
You should change for :+1: 

`$this->assertContains('Password encoding succeeded', $this->passwordEncoderCommandTester->getDisplay());`